### PR TITLE
Rails 5 update foundation

### PR DESCRIPTION
**Pull request by sineade**
🐘 
This branch is specifically for updating gatekeeper while it is pointing to the rails 5 branch of ama_layout. 

This updates to foundation 6.3.1.0

### CODE REVIEW

**Comment by MathieuGilbert** [comment]
:elephant: :elephant: :elephant:

**Comment by MathieuGilbert** [comment]
💩 Oops, Travis is failing

**Comment by MathieuGilbert** [comment]
:elephant: :elephant: :elephant:

File: lib/ama_layout/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module AmaLayout
-  VERSION = '6.1.0.pre'
+  VERSION = '6.2.0.pre'
```
Comment (kaytcampbell) [inline lib/ama_layout/version.rb]: is 6.3 not compatible with rails 5?

Comment (kaytcampbell) [inline lib/ama_layout/version.rb]: oh wait this is not foundation. disregard 😆

**Pull request merged**